sklearn/_config.py

```diff
@@ -53,6 +53,9 @@ def set_config(assume_finite=None, working_memory=None,
         all the non-changed parameters.
 
         .. versionadded:: 0.21
+    See also
```

Comment:
```suggestion
    See Also
```

Reply:
Currently we mostly use the non capitalized version,
```
rg "See also" sklearn  | wc -l                                                           
203
rg "See Also" sklearn  | wc -l                                                            
26
```
we also don't always capitalize the first letters in titles e.g. in [the user guide](https://60731-843222-gh.circle-artifacts.com/0/doc/user_guide.html).

I'm not sure which one is more correct if that is to be changed maybe better to keep it consistent?

Reply:
Interesting. I suggested this because it was [not rendering correctly before](https://60731-843222-gh.circle-artifacts.com/0/doc/modules/generated/sklearn.set_config.html#sklearn.set_config). Now [it is](https://60739-843222-gh.circle-artifacts.com/0/doc/modules/generated/sklearn.set_config.html#sklearn.set_config).

Reply:
Yes, a new line was missing.